opencsp/common/lib/tool/hdf5_tools.py

```diff
@@ -242,3 +245,36 @@ def unzip(hdf5_path_name_ext: str, destination_dir: str, dataset_format='npy'):
             np.savetxt(dataset_path_name + ".csv", squeezed, delimiter=",")
 
     return hdf5_dir
+
+
+class HDF5_SaveAbstract(ABC):
+    """Abstract class for saving to HDF5 format"""
+
+    @abstractmethod
+    def save_to_hdf(self, file: str, prefix: str = '') -> None:

```

Comment:
I really like this interface! Having a "prefix" should really clean up how we interact with HDF5 files.